site/content/resources/support.md

```diff
@@ -5,20 +5,51 @@ layout: page
 
 This document describes which versions of Contour are supported by the Contour team.
 
-## Stable release
+## Supported releases
 
-Only the latest stable release is supported.
+Contour is changing both to quarterly releases and three supported releases.
 
+The first Contour version covered by the quarterly release cadence will be Contour v1.20, scheduled for late October 2021.
```

Comment:
```suggestion

```

Reply:
I don't understand why you want to remove this? Right now, we need to ensure that during this transition period, this is clearly explained for people who are using Contour. That's the point of all of this present-tense language. This will not live forever, once we are fully moved over to the new release schedule, then I will update this again.

As of 1.19, we are *not* releasing quarterly or supporting three versions. Saying otherwise is incorrect, and confusing.